src/dotnet/Chat.Service/Conversations.cs

```diff
@@ -15,4 +16,23 @@ public virtual async Task<Conversation[]> GetTile(Session session, ChatId chatId
 
         return await Backend.GetTile(chatId, idTileRange, cancellationToken).ConfigureAwait(false);
     }
+
+    public virtual Task<Conversation?> Get(ConversationId conversationId, CancellationToken cancellationToken)
```

Comment:
In public API we always pass Session as a parameter in order to control who can access the data.

Reply:
Use session to verify if user should have access to the conversation. See GetTile method as an example.